src/style.css

```diff
@@ -1 +1,100 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+body {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  display: flex;
+  justify-content: center; /* Центрування форми по горизонталі */
+  align-items: center;
+  width: 100%; /* Ширина для flex-контейнера */
+  margin-top: 20px;
+}
+
+.input__wrapper {
+  position: relative;
+  width: 100%; /* Задаємо 100% ширину обгортки */
+  display: flex;
+  align-items: center;
+}
+
+.input__icon__1 {
+  position: absolute;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.input__icon__2 {
+  position: absolute;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.input__first {
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  font-weight: 300;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background-color: white;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir-Book, sans-serif;
+}
+
+.input__first:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.input__first:focus {
+  background: linear-gradient(180deg, 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+  text-shadow: 0 4px 4px 0 #00000040;
+  outline: none;
+  color: #3d4e61;
+  font-family: Avenir-Heavy, sans-serif;
+}
+
+.input__second {
+  padding-left: 33px;
+  width: 100%;
+  height: 42px;
+  font-size: 14px;
+  font-weight: 400;
+  background-color: white;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir-Book, sans-serif;
```

Comment:
The font-family declaration here is redundant as well since it's the same as the one set for the `body`. You can remove it unless you need a different font for this specific element.